test/integration/stats/data/requests_to_api_test.rb

```diff
@@ -19,7 +21,7 @@ def setup
   test 'usage with access token' do
     member = FactoryBot.create(:member, account: @provider_account, admin_sections: ['monitoring'])
     token  = FactoryBot.create(:access_token, owner: member, scopes: ['stats'])
-    params = { period: 'day', metric_name: 'hits', access_token: token.value }
+    params = { period: 'day', since: Time.utc(2020, 8, 1).to_date, metric_name: 'hits', access_token: token.value }
```

Comment:
We used to have tests that suggest an expected 200 response for API requests missing the `since` param, even though this is documented as a required param 🤔: https://github.com/3scale/porta/blob/ff0b72315fcd58397aeac04aa254ad6c66376f52/app/controllers/stats/data/base_controller.rb#L31

Reply:
don't worry, it was probably unintentional at the time OR it just changed over time 🤷‍♀️ the point is that we need it 👍

Reply:
I know why already. I'll fix it.

Reply:
It's because of the escape condition at https://github.com/3scale/porta/blob/ff0b72315fcd58397aeac04aa254ad6c66376f52/app/lib/stats/views/usage.rb#L135